src/WebDav.Client/IWebDavClient.cs

```diff
@@ -400,5 +401,13 @@ public interface IWebDavClient : IDisposable
         /// <param name="parameters">Parameters of the UNLOCK operation.</param>
         /// <returns>An instance of <see cref="WebDavResponse" />.</returns>
         Task<WebDavResponse> Unlock(Uri requestUri, UnlockParameters parameters);
+
+        /// <summary>
+        /// Use WebDav SEARCH Method.
+        /// </summary>
+        /// <param name="requestUri">Request uri.</param>
+        /// <param name="parameters">SEARCH parameters.</param>
+        /// <returns></returns>
+        Task<WebDavResponse> Search(Uri requestUri, SearchParameters parameters)
```

Comment:
Missing semicolon, doesn't build.

Reply:
Sorry, my mistake, copied it in a hurry between two projects.